src/demo-app/sidenav/sidenav-demo.html

```diff
@@ -48,6 +48,18 @@
   </div>
 </md-sidenav-container>
 
+<h2>Responsive Drawer</h2>
+
+<md-sidenav-container class="demo-sidenav-container" breakpointWidth="600" breakpointChangeMode>
```

Comment:
I'm not so sure about this API, it doesn't seem to give much flexibility. You can't control what modes it switches between and you can't disable the auto-close/open feature